readme.md

```diff
@@ -36,6 +36,13 @@ provided as a `"js"` parameter.
 + A result of a custom async JavaScript invocation (`"js"` and `"jsOn"` parameters) is returned in the response as `"result"`.
 + A result of the PDF generation (`"takePdfSnapshot"` parameter) is returned in the response as `"pdfSnapshot"`.
 
+> [!NOTE]  
+> There are utility functions available in the `window` object which can be used in the custom JavaScript code: 
+
+- `async pageTrueLoad ({
+    idleTimeout = 2000, resolveBeforeRenderingTimeout = 2000
+  } = {})` - a promise that resolves when the page is fully loaded (including delayed requests).
```

Comment:
I'd rephrase `resolves when there are no more pending network requests, but resolveBeforeRenderingTimeout before the global timeout`

Reply:
BTW maybe a better name is `await webrender.pendingRequests()` for this function
Up to you